torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -208,6 +208,37 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
     checkAndStartRecordFunction(frames.back(), stack);
   }
 
+#if defined(__GNUC__) || defined(__clang__)
+#define USE_COMPUTED_GOTO
```

Comment:
Nit:
Should we use a more specific name like JIT_USE_COMPUTED_GOTO and `undef` it at end of this logic block?